layouts/auto_generated_product.tpl

```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
-{% include "template-variables" %}
+{%- assign product_page = true -%}
+{%- include "template-settings" -%}
+{%- include "template-variables" -%}
 
 <html class="{% if editmode %}editmode{% else %}public{% endif %}" lang="{{ page.language_code }}">
 
 <head prefix="og: http://ogp.me/ns#">
   {% include "html-head" %}
-  {% include "template-styles" %}
+  {% include "template-styles" body_bg_key: template_settings.product.body_bg.key %}
```

Comment:
It's not needed to by pass `body_bg_key` here as attribute, because of `body_bg_key` is assigned allready in  `template-variables`

https://github.com/Voog/design-boilerplate/blob/0d7817d23cab75efa62613e5a220acb151312a8b/components/template-variables.tpl#L7

```suggestion
  {% include "template-styles" %}
```